rotkehlchen/tests/external_apis/test_yearn.py

```diff
@@ -0,0 +1,53 @@
+import datetime
+
+from freezegun import freeze_time
+
+from rotkehlchen.chain.ethereum.modules.yearn.utils import query_yearn_vaults
+from rotkehlchen.chain.evm.types import string_to_evm_address
+from rotkehlchen.constants.timing import WEEK_IN_SECONDS
+from rotkehlchen.globaldb.handler import GlobalDBHandler
+from rotkehlchen.types import YEARN_VAULTS_V2_PROTOCOL, ChainID, GeneralCacheType
+
+
+def test_yearn_api(database):
```

Comment:
Does this query an external api and take a long time? OR just short time?

Reply:
Is really short time. 1 second in the whole test in my computer

Reply:
This too will need to be mocked eventually. We need to mock as many tests as possible. If it's repeatable it is mockable.